declarations/Finya.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Finya",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.finya.de/Legal/terms/",
+      "select": [
+        "main"
+      ]
+    },
+    "Privacy Policy": {
+      "fetch": "https://www.finya.de/Legal/privacy/",
+      "select": [
+        "main"
+      ],
+      "remove": [
+        "main > div:last-child"
+      ]
+    },
+    "Imprint": {
+      "fetch": "https://www.finya.de/Legal/about/",
+      "select": [
+        {
+          "startAfter": "main > h2:nth-of-type(1)",
+          "endBefore": "main > h2:nth-of-type(2)"
+        }
+      ]
```

Comment:
I understand your intention but I'm wondering whether it wouldn't be better to save the whole document in view of the selectors. What do you think too?

Reply:
Including the Mitgliedschaften section would mean also including the images/image links?

Yeah, I guess that's fine. It's their Imprint, what they define it to be, so we can reasonably take it as what *they* think it should be rather than *us* deciding.

Reply:
> Including the Mitgliedschaften section would mean also including the images/image links?

The images here seem significant, so I think we can include the whole `main` block.

Reply:
```suggestion
      "select": [
         "main"
      ]
```